dts/bindings/debug/arm,armv7m-itm.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: ARMv7 ITM
```

Comment:
Can you please expand the acronym ITM in these descriptions, so readers have a better chance of figuring out what's going on here?

Reply:
Updated with a more detailed description